### ISSUE TITLE
Implementation of prettier for public dir

### ISSUE THREAD

**Issue by vatsalsinghkv** [ufosc/Club_Website_2#122]
- I noticed there wasn't any styling tool for css files
- Prettier would really be good for have same styles and syntax for everyone who's contributing

**Comment by zeim839** [comment]
That'd be very useful, would you like to be assigned to this issue?

Also note that the prettier module should be configured to only target CSS files, because JS code is already handled by standardJS. Once this is done, the `npx prettier --write .` command should be added to the "fix" script in package.json, and the `npx prettier --check .` command to the "lint" script.

**Comment by vatsalsinghkv** [comment]
Yes please assign this to me I'd love to implement it, sure I'll make sure files like html and css are styled by prettier

**Comment by zeim839** [comment]
> would it be okey if prettier formats .md and .yml files too?

Yes, that's fine.

**Comment by vatsalsinghkv** [comment]
please check #125
